### PR TITLE
Align Asciidoctor Layout and Styling to Antora

### PR DESCRIPTION
**Pull request by julioest**
### Description
This PR aligns the Asciidoctor layout and styling with our latest Antora design.

### Issues:
- ~The table elements overflow when the container becomes narrower. (Still working on a fix)~

### CODE REVIEW

**Comment by julioest** [comment]
@vinniefalco Here's a preview of the latest
image

**Comment by vinniefalco** [comment]
Exciting :)

File: boostlook.css [hunk 1]
```diff
@@ -137,3 +137,70 @@ padding: 0.25em 0.55em;
...
+
+.boostlook #toc.toc2 {
+  padding-left: 0;
+}
+
+.boostlook #content {
+  overflow-x: auto;
+}
+
+@media screen and (min-width: 768px) {
```
Comment (vinniefalco) [inline boostlook.css:190]: This should be using container query

Comment (julioest) [reply boostlook.css:194]: Tried making it work with the @container query, I'm not too familiar with it btw. I'm going to revisit how we can leverage that query in future PRs.

**Pull request merged**